src/gui/dispersionreticle/translations/translations_zh_cn.json

```diff
@@ -35,33 +35,33 @@
   "focusedReticle.header": "最小落弹环",
   "focusedReticle.body": [
     "在显示标准瞄准环的同时，显示出最小的缩圈大小。\n",
-    "Uses builtin game logic to do so.\n",
-    "Other mods might influence the look of this reticle.\n"
+    "使用游戏机制实现。\n",
+    "同时使用其他类似模组可能影响显示效果。\n"
   ],
   "focusedReticle.note": [
     "在打开了插件中的服务器准星选项时最小缩圈将显示在客户端准星上。\n\n"
   ],
-  "focusedReticleExtended.label": "Focused reticle extended",
-  "focusedReticleExtended.header": "Focused reticle extended",
+  "focusedReticleExtended.label": "最小落弹环外观配置",
+  "focusedReticleExtended.header": "最小落弹环外观配置",
   "focusedReticleExtended.body": [
-    "Adds configurable reticle displaying fully-focused dispersion to in-game reticle.\n",
-    "Other mods does not have influence on the look of this reticle."
+    "这里可以配置最小落弹环的外观。\n",
```

Comment:
Extended reticles are independent from the basic ones; for example `focused reticle` and `focused reticle extended` are two independent reticles.

They only differ by how they are implemented:
* basic ones uses same logic as for default reticles, but at the cost of limited configuration
* extended ones are rendered by my logic, so with various configuration

Could you update it to match that description (for extended hybrid and extended server as well)?

Reply:
As I have tested in game , the extended one always be at the same place as basic one. As the final result are basically like adding visual effect( T mark , pentagon, width , height , thickness, etc) , so I translated as if it was changing properties of the reticle.  Maybe this will not make user too confuse about too many different reticles. 

The word "hybrid" is something vague in Chinese,  "network delay" is easier to understand in context of network gaming.

Reply:
Yeah, as a result, there might me some confusion that one extends another, but in reality they are completely independent.

I wanted to keep them separated due to different rendering logic and due to this the have some important differences:
* basic ones are rendered exactly the same way game renders normal reticles
* basic ones "catches" changes from other mods, but I cannot customize them
* extended reticles are rendered 100% by me, so I can do anything with them

Currently you can enable "both versions" of the same reticle type, what isn't really useful, but I haven't got any better idea for handling it yet for now